### ISSUE TITLE
Patterns in common shared directory or in separate subdirectories

### ISSUE THREAD

**Issue by ahovgaard** [ahovgaard/TiPL-kleenex-approx#1]
In lack of a better communication channel, I'll just put this here...

As stated in the `README`, the files in the `patterns` directory is for all the four tools. However, all four tools require quite different input formats, so is it best to have the differently formatted patterns in each subdirectory or to write scripts to transform these shared patterns into the appropriate formats?

**Comment by ahovgaard** [comment]
```
TGCAAGCGTTAAT[2,0,0]
```
```
main := (/TGCAAGCGTTAAT/<2> "+" | ~/./)*
```
```
nrgrep -N -c -k "2s" "TGCAAGCGTTAAT"
```
etc.

**Comment by Ginko-X** [comment]
Yes I realized this problem. I planed to discuss it when we group meet. 
In my opinion, the dir patterns is just to show the original patterns we test with. It's true that we need to change the format when we test them in different tools. Because we work separately, it's not so convenient to check what other group members' patterns for each tool.  
Therefore, this is just for convenience, not quite necessary.
